File: horizontalpicker/src/main/java/com/jhonnyx/horizontalpicker/HorizontalPickerListener.java
package com.jhonnyx.horizontalpicker;

/**
 * Created by jhonn on 02/03/2017.
 */
public interface HorizontalPickerListener {
    void onStopDraggingPicker();
    void onDraggingPicker();
    void onDateSelected(Day item);
}

File: horizontal-picker/src/main/java/com/github/jhonnyx2012/horizontalpicker/HorizontalPickerListener.kt
package com.github.jhonnyx2012.horizontalpicker

/**
 * Created by jhonny on 02/03/2017.
 */
interface HorizontalPickerListener {
    fun onStopDraggingPicker()
    fun onDraggingPicker()
    fun onDateSelected(item: Day)
}